packages/bodiless-components/src/Chameleon/asBodilessChameleon.tsx

```diff
@@ -25,13 +25,16 @@ import withChameleonButton from './withChameleonButton';
 import applyChameleon from './applyChameleon';
 import withChameleonContext from './withChameleonContext';
 
-const withDeleteNodeOnUnwrap = <P extends object>(Component: ComponentType<P> | string) => {
+const withDeleteNodeOnUnwrap = (
+  path: string,
```

Comment:
I think rename this to 'nodeKey' - and please add jsdocs for this exported function.

Reply:
Also - since you're changing the signature of this function, youi'll have to update its usage elsewhere (primarily on the list2 example page and in the new List docs, but i'd do a global search..

Reply:
Done. Usage and Documentation is also updated